hub/subscribe.go

```diff
@@ -202,3 +233,53 @@ func (h *Hub) cleanup(s *Subscriber) {
 	}
 	h.uriTemplates.Unlock()
 }
+
+func (h *Hub) dispatchSubscriptionUpdate(topics, encodedTopics []string, connectionID string, claims *claims, active bool, address string) {
+	if !h.config.GetBool("dispatch_subscriptions") {
+		return
+	}
+
+	for k, topic := range topics {
+		connection := &subscription{
+			ID:      "https://mercure.rocks/subscriptions/" + encodedTopics[k] + "/" + connectionID,
+			Type:    "https://mercure.rocks/Subscription",
```

Comment:
Do you think that the hostname could be made configurable in the future?

Reply:
It's not a host name, it's a RDF type. It's similar to Schema.org type: their IDs never change. TL;DR: it's fixed on purpose.

Reply:
okay then.